modules/react/menu/lib/MenuItem.tsx

```diff
@@ -40,7 +48,56 @@ export interface MenuItemProps {
   'aria-disabled'?: boolean;
 }
 
-const StyledItem = styled(Box.as('button'))<StyledType>(
+export interface StyledMenuItemProps {
+  /**
+   * The function called when the DeprecatedMenuItem is clicked. If the item is a child of the DeprecatedMenu component, this callback will be decorated with the onSelect and onClose DeprecatedMenu callbacks. This callback will not fire if the item is disabled (see below).
+   */
+  onClick?: (event: React.MouseEvent) => void;
+  /**
+   * The unique id for the DeprecatedMenuItem used for ARIA attributes. If the item is a child of the `DeprecatedMenu` component, this property will be generated and overridden.
+   */
+  id?: string;
+  /**
+   * The icon of the DeprecatedMenuItem. This icon is displayed before what you supplied for the children.
+   */
+  icon?: CanvasSystemIcon;
+  /**
+   * The secondary icon of the DeprecatedMenuItem. This icon is displayed after what you supplied for the children.
```

Comment:
Is this still correct? `DeprecatedMenuItem`? The one in main is not deprecated

Reply:
That's a copy over mistake. I'll fix it